src/scripts/core/models/token.ts

```diff
@@ -44,7 +44,7 @@ export class Token extends Container {
         eventEmitter.emit('clickCheck', this);
```

Comment:
comment about the above constructor, instead of having

 this.skIndex = Math.round(Math.random() * (9 - 1) + 1);;
        this.skin = new Spine(Assets.get('symbols').spineData);
        this.skin.skeleton.setSkinByName(`${this.skIndex}`);

you could just call this.shuffleSkin(), any future changes to setting random skins would mean only having to change that in one location